portfolio/src/main/java/com/google/sps/servlets/CreatePuzzleServlet.java

```diff
@@ -64,14 +64,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     Puzzle puzzle = new Puzzle();
 
     puzzle.setImageUrl(uploadedFileUrl);
+    puzzle.setDifficulty(difficulty);
```

Comment:
Here is where you would do `Difficulty.valueOf(difficulty)` and that will give you the enum.